app/src/test/java/org/oppia/android/app/testing/options/AppLanguageFragmentTest.kt

```diff
@@ -104,6 +112,19 @@ class AppLanguageFragmentTest {
     Intents.release()
   }
 
+  @Test
+  fun testAppLanguageActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      createAppLanguageActivityIntent(
+        "English"
+      )
+    )
+    val title = activityTestRule.activity.title
+    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+    // correct string when it's read out.
+    Truth.assertThat(title).isEqualTo(context.getString(R.string.app_language_activity_title))
```

Comment:
Use direct import and write it as 
`assertThat(title).isEqualTo(context.getString(R.string.app_language_activity_title))`

Reply:
done